package.json

```diff
@@ -2,13 +2,15 @@
   "name": "fxa-auth-db-mysql",
   "version": "0.0.0",
   "description": "MySQL backend for the fxa-auth-db-server",
-  "main": "index.js",
+  "main": "bin/server.js",
   "repository": "mozilla/fxa-auth-db-mysql",
   "bin": {
     "fxa-auth-db-mysql": "db_patcher.js"
```

Comment:
I don't think we need to export an executable binary, and even if we do, it doesn't seem right to export just the patcher like this.